tests/autotuner/test_autotuner_mm_bf16.py

```
@@ -0,0 +1,159 @@
+    out = mm_bf16(a, b, backend=backend)
+    assert out.isfinite().all()
+
+    # 3) The tuned bucket entry must be reused for the non-bucket M.
+    if backend == "cudnn":
+        runner = _cudnn_gemm_bf16_runner(
+            is_a_k_major=a.stride(-1) == 1,
+            is_b_k_major=b.stride(-2) == 1,
+        )
+    else:
+        runner = get_mm_bf16_cublaslt_module().cublaslt_bf16_gemm_runner()
+    is_cache_hit, runner_id, tactic, stored_profile = _search_bf16_gemm_cache(
```

Ruff (0.15.20)

[warning] Unpacked variable `runner_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)